src/PrestaShopBundle/Form/Admin/Sell/Catalog/AttributeType.php

```diff
@@ -0,0 +1,153 @@
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param bool $isMultistoreEnabled
+     * @param AttributeGroupRepository $attributeGroupRepository
+     * @param int $contextShopId
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        AttributeGroupRepository $attributeGroupRepository,
+        AttributeGroupDataProvider $attributeGroupDataProvider,
+        int $contextShopId,
+        int $contextLangId
+    ) {
+        parent::__construct($translator, $locales);
+
+        $this->attributeGroupRepository = $attributeGroupRepository;
+        $this->contextShopId = $contextShopId;
+        $this->attributeGroupDataProvider = $attributeGroupDataProvider;
+        $this->contextLangId = $contextLangId;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $attributeGroupId = $options['data']['attribute_group'];
+        $attributeGroup = $this->attributeGroupRepository->get(
+            new AttributeGroupId($attributeGroupId),
+            new ShopId($this->contextShopId)
+        );
+        $builder
+            ->add('attribute_group', ChoiceType::class, [
+                'label' => $this->trans('Attribute group', 'Admin.Catalog.Feature'),
+                'help' => $this->trans('The way the attribute\'s values will be presented to the customers in the product\'s page.', 'Admin.Catalog.Help'),
+                'choices' => $this->attributeGroupDataProvider->getAttributeGroupChoices(
+                    new ShopId($this->contextShopId),
+                    new LanguageId($this->contextLangId)
+                ),
+            ])
+            ->add('value', TranslatableType::class, [
+                'type' => TextType::class,
+                'label' => $this->trans('Value', 'Admin.Global'),
+                'options' => [
+                    'constraints' => [
+                        new TypedRegex([
+                            'type' => TypedRegex::TYPE_CATALOG_NAME,
+                        ]),
+                    ],
+                ],
+                'help' => $this->trans('Your internal name for this attribute.', 'Admin.Catalog.Help')
+                    . '&nbsp;' . $this->trans('Invalid characters:', 'Admin.Notifications.Info')
+                    . ' ' . TypedRegexValidator::CATALOG_CHARS,
+            ]);
+        if ($attributeGroup->group_type === AttributeGroupType::ATTRIBUTE_GROUP_TYPE_COLOR) {
+            $builder->add('color', ColorType::class, [
+                'label' => $this->trans('Color', 'Admin.Global'),
+                'required' => false,
+            ])->add('texture', FileType::class, [
+                'label' => $this->trans('Texture', 'Admin.Global'),
+                'required' => false,
+            ]);
+        }
+
+        $builder->add('shop_association', ShopChoiceTreeType::class, [
```

Comment:
I think this field is only needed when multistore is activated But it can be improved in another PR I guess